API.md

```diff
@@ -169,6 +169,24 @@ Draw
 
 ---
 
+### `.set(Object: featureCollection) -> [featureId, featureId]`
+
+This function takes a featureCollection and performs the required delete, create and update actions internally to make Draw represent this change. Effectively this is the same as `Draw.deleteAll()` followed by `Draw.add(featureCollection)` except that it doesn't effect performance as much.
+
+Example:
+
+```js
+var ids = Draw.set({type: 'FeatureCollection', features: [{
+  type: 'Feature',
+  properties: {},
+  id: 'example-id',
+  geometry: { type: 'Point', coordinates: [0, 0] }
+}]});
+// => ['example-id']
+```
+
```

Comment:
Answers based on reading the code below (which we should add to docs at some point — please correct me if they're wrong):

Yes, it only accepts FeatureCollections.

It compares features based on `id`. So any feature whose `id` matches a feature in the new collection will be replaced (completely, not merged) with the collection's feature.